### PR TITLE
fix glyphs to use regular charcode

### PR DESCRIPTION
**Pull request by ThyWoof**

### CODE REVIEW

File: ModKit/UI/UI+Elements.cs [hunk 1]
```diff
@@ -4,12 +4,12 @@
 namespace ModKit {
     public static partial class UI {
 
-        public const string ChecklyphOn = "<color=green><b>✔</b></color>";
```
Comment (cabarius) [inline ModKit/UI/UI+Elements.cs:7]: please don't replace these values.  Please provide and alternate set of glyphs and put it behind a settings toggle.

Comment (dark0dave) [reply ModKit/UI/UI+Elements.cs:7]: From
https://github.com/cabarius/ToyBox/issues/499
https://github.com/cabarius/ToyBox/issues/483

@cabarius could you point us in the right direction for that? Appologies, its not super clear where to do so.

Comment (dark0dave) [reply ModKit/UI/UI+Elements.cs:7]: We could create annother class, is that the intent? CharcodeUI ?

**Comment by cabarius** [comment]
Took care of this with the other PR and made some improvements.

**Pull request closed without merging**